apps/ledger-live-mobile/src/newArch/features/Web3Hub/utils/api/manifests.ts

```diff
@@ -36,8 +37,19 @@ export const fetchManifests: (
 ) => QueryFunction<LiveAppManifest[], string[], number> =
   (category, search) =>
   async ({ pageParam }) => {
+    const url = new URL(`${URL_ORIGIN}/api/v2/apps`);
```

Comment:
We introduced react-query on both LLM & LLD, so feel free to use it if you need it.

Reply:
Nice thx for the heads up

Reply:
Actually I'm already using it here the `QueryFunction ` for example comes from it, so you would it help for this ?